store/views.py

```diff
@@ -54,15 +105,32 @@ def viewLoanedBooks(request):
 @csrf_exempt
 @login_required
 def loanBookView(request):
+    book_id = request.POST.get('bid')
+    count = BookCopy.objects.filter(Q(book=Book.objects.get(id=book_id)) & Q(status=True)).count()
```

Comment:
This is not a good way to call ORM queries. This will run two queries on database, one to get the book and another to filter it.
Better, you should've used: `book=book_id`